service_configuration_lib/spark_config.py

```diff
@@ -46,7 +47,7 @@
     'spark.executorEnv.SPARK_EXECUTOR_DIRS',
     'spark.hadoop.fs.s3a.access.key',
     'spark.hadoop.fs.s3a.secret.key',
-    'spark.hadoop.fs.s3a.session.token'
+    'spark.hadoop.fs.s3a.session.token',
```

Comment:
need to figure out how to bump this version since this should have been identified as a syntax error